lib/api/api_connect.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:http/http.dart' as http;
+import 'dart:async';
+import 'dart:convert';
+import 'package:Solon/api/message.dart';
+import 'package:Solon/api/user.dart';
+import 'package:Solon/api/proposal.dart';
+import 'package:Solon/api/comment.dart';
+import 'package:Solon/api/event.dart';
+import 'package:Solon/api/forumpost.dart';
+import 'package:Solon/api/vote.dart';
+import 'dart:io';
+import 'package:flutter/services.dart';
+import 'package:intl/intl.dart';
+
+class APIConnect {
+  static String _url = "https://api.solonedu.com";
+
+  static Future<String> loadHeader() async {
+    return await rootBundle.loadString('assets/secret');
+  }
+
+  static Future<Message> connectRoot() async {
+    final response = await http.get(_url);
+    int status = response.statusCode;
+    return status == 200
+        ? Message.fromJson(json.decode(response.body)['message'])
+        : throw Exception('data not found');
```

Comment:
we can probably have better exception messages.

Reply:
check out my slightly better exceptions msgs